### PR TITLE
Add monthly update for April 2023.

### PR DESCRIPTION
**Pull request by kgpai**

### CODE REVIEW

**Comment by kgpai** [comment]
Heres gist of commits : https://gist.github.com/kgpai/22332e6fc7621479b96119e5d9703fda

File: velox/docs/monthly-updates/april-2023.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Presto Functions
+================
+
+ * Add :func:`timezone_minute`,   :func:`timezone_hour` functions.
+ * Add :func:`date`,   :func:`random()`,  :func:`sequence`,  :func:`from_base64url` functions.
+ * Add support for DECIMAL comparison functions.
+ * Add support for DECIMAL :func:`abs` and :func:`negate` functions.
+ * Add :func:`first_value` and :func:`last_value` window functions.
+ * Add :func:`from_big_endian_32`,  :func:`from_big_endian_64`,  :func:`to_big_endian_32`,  :func:`to_big_endian_64` functions.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/april-2023.rst:28]: Move this item next to other functions. Make sure functions are grouped according to some pattern.

File: velox/docs/monthly-updates/april-2023.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+Spark Functions
+===============
+
+ * Add :spark:func:`substring`,   :spark:func:`might_contain` functions.
+ * Add :spark:func:`unix_timestamp`,  :spark:func:`to_unix_timestamp`function.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/april-2023.rst:35]: function -> functions

File: velox/docs/monthly-updates/april-2023.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ * Add :spark:func:`bit_xor` aggregate function.
+
+
+Hive Connector
+==============
+
+ * Add support for pushing down deletes into reader.
+ * Add Metalake support to PrismConnector.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/april-2023.rst:43]: Is PrismConnector open source? I assume not. Let's remove this item.

File: velox/docs/monthly-updates/april-2023.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+ * Fix map key pruning when all keys are filtered out.
+
+
+Performance and Correctness
+===========================
+
+ * Enable coalesce in expression fuzzers.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/april-2023.rst:50]: fuzzers -> fuzzer

File: velox/docs/monthly-updates/april-2023.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+ * Fix error vector handling in SwitchExpr
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/april-2023.rst:51]: This is a bit cryptic. Can you clarify?

File: velox/docs/monthly-updates/april-2023.rst [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+ * Fix memory leak in expression eval.
+ * Fix :spark:func:`map` function.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/april-2023.rst:53]: Delete or move this to Spark functions section.

File: velox/docs/monthly-updates/april-2023.rst [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+ * Safer decoding for ConstantVector<bool>.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/april-2023.rst:54]: Reword to start with a verb.

File: velox/docs/monthly-updates/april-2023.rst [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+
+Build System
+============
+
+ * Add support to alert on changes in function signatures.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/april-2023.rst:60]: Do we have some details about this ? Let's add a link.

**Review by mbasmanova — COMMENTED** [review]
@kgpai Thank you for putting this together. Looks good % a couple of nits.

CC: @majetideepak

File: velox/docs/monthly-updates/april-2023.rst [hunk 9]
```diff
@@ -0,0 +1,75 @@
+********************
+April 2023 Update
+********************
+
+
+Documentation
+=============
+
+ * Add :doc:`documentation on UnsafeRow serialization <../develop/unsaferow>`.
```
Comment (majetideepak) [inline velox/docs/monthly-updates/april-2023.rst:9]: Absolute paths `/develop/unsaferow>` are better. When this page gets moved to the 2023 folder next year, this does not have to change.

File: velox/docs/monthly-updates/april-2023.rst [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+ * Add support for companion function generation for aggregations.
+ * Add support to suppress errors for unused arguments of null-propagating functions.
+
+
+Presto Functions
+================
+
+ * Add :func:`timezone_minute`,   :func:`timezone_hour` functions.
+ * Add :func:`date`,   :func:`random()`,  :func:`sequence`,  :func:`from_base64url` functions.
+ * Add support for DECIMAL comparison functions.
```
Comment (majetideepak) [inline velox/docs/monthly-updates/april-2023.rst:25]: This should probably be "Implement DECIMAL comparisons as vector functions".
Decimal comparisons were already implemented as simple functions. The recent change is that they have been re-implemented as vector functions.

File: velox/docs/monthly-updates/april-2023.rst [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+ * Add :spark:func:`bit_xor` aggregate function.
+
+
+Hive Connector
+==============
+
+ * Add support for pushing down deletes into reader.
+ * Add Metalake support to PrismConnector.
+ * Fix map key pruning when all keys are filtered out.
+
```
Comment (Yuhta) [inline velox/docs/monthly-updates/april-2023.rst:45]: We can also add #4236

**Review by xiaoxmeng — APPROVED** [review]
@kgpai thanks for the update!

**Pull request closed without merging**